tracks/python/exercises/pig-latin/pig-latin/mentoring.md

```diff
@@ -0,0 +1,42 @@
+# Pig Latin
+
+
+## Common Areas of Improvement
+When mentoring students on the Pig Latin exercise, it's essential to focus on common areas where they might struggle. Here are some key points to keep in mind:
+
+1. Understanding the Rules Clearly
+    - Rule Comprehension: Ensure that students fully understand each of the Pig Latin translation rules before diving into coding. Misunderstanding the rules often leads to incorrect implementations.
+    - Edge Cases: Highlight the importance of edge cases such as words starting with "xr" or "yt", and those containing "qu".
+2. String Manipulation Skills
+    - Splitting and Joining Strings: Students should be comfortable using functions to split sentences into words and join words back into sentences.
+    - String Slicing: Emphasize the importance of correctly slicing strings to rearrange parts of the word as per the rules.
+    - Prefix Checking: Encourage using built-in string functions like starts with to simplify rule checks.
+3. Iterating Over Strings
+    - Looping Over Characters: Ensure students understand how to iterate over characters in a string, especially when finding the first vowel or handling specific patterns.
+    - Range-based Loops: Encourage using range-based loops or list comprehensions for readability and efficiency.
+4. Condition Handling
```

Comment:
Is there any specific examples in Pig Latin where this applies? The second point sounds pretty generic and not specific to this exercise.